src/main/java/adris/altoclef/trackers/InventoryTracker.java

```diff
@@ -491,9 +575,10 @@ public MiningRequirement getCurrentMiningRequirement() {
 
             // Repeat this collection "count" number of times.
             for (int i = 0; i < count; ++i) {
-                boolean foundMatch = false;
+                boolean matchNotFound = true;
                 //noinspection SpellCheckingInspection
                 itemsearch:
+                // TODO: 2021-05-22 please never use these
```

Comment:
To not use gotos, yeah definitely. But we're breaking out of a nested for loop here, thought that was an acceptable use case that can be clean. Other option would be to hold a boolean for this (which maybe I already do? Might work out)

Reply:
Yeah, basically every single linter for Java ever will throw errors up the wazoo for these guys.